contracts/QuestChain.sol

```diff
@@ -455,17 +445,16 @@ contract QuestChain is
      * @dev internal function to update status of quest to review
      * @param _questId identifier of quest
      */
-    function _submitProof(uint256 _questId)
-        internal
-        whenQuestNotPaused(_questId)
-        validQuest(_questId)
-    {
+    function _submitProof(uint256 _questId) internal validQuest(_questId) {
+        require(!questDetails[_questId].paused, "QuestChain: quest paused");
         require(
             _questStatus[_msgSender()][_questId] != Status.pass,
             "QuestChain: already passed"
         );
 
-        _questStatus[_msgSender()][_questId] = Status.review;
+        questDetails[_questId].skipReview
+            ? _questStatus[_msgSender()][_questId] = Status.pass
+            : _questStatus[_msgSender()][_questId] = Status.review;
```

Comment:
@dan13ram quests with skipReview will pass directly, but there is no particular event for this. Can this be an issue for the subgraph? It will only emit `QuestProofsSubmitted` like any other quest.